lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -3536,17 +3391,8 @@ namespace Js
         // so that the update is in sync with profiler
         JavascriptLibrary* library = objectConstructor->GetLibrary();
         ScriptContext* scriptContext = objectConstructor->GetScriptContext();
-        int propertyCount = 17;
-        if (scriptContext->GetConfig()->IsES6SymbolEnabled())
-        {
-            propertyCount += 1;
-        }
-        if (scriptContext->GetConfig()->IsES6ObjectExtensionsEnabled())
-        {
-            propertyCount += 2;
-        }
 
-        typeHandler->Convert(objectConstructor, mode, propertyCount);
+        typeHandler->Convert(objectConstructor, mode, 20);
```

Comment:
I considered that but all these Initialize functions use unnamed constants, and when you look at the parameter name its obvious what the value is for.  I decided it wasn't worth bothering with.